### PR TITLE
Implementation of the dummy distribution federate

### PR DESCRIPTION
**Pull request by AadilLatif**
models a voltage source, an unbalanced distribution line, and unbalanced voltage-dependent loads on the three phases. A simple forward-backward sweep is used to converge and return the sum of power across all phases to the GridlabD simulator

### CODE REVIEW

**Review by nightlark — COMMENTED** [review]
It looks like the python script file name has a typo -- `dirtrbution.py` instead of `distribution.py`

File: distribution1/dirtrbution.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+def collect_data(data_list, idx, time, value):
+    data_list[idx][0].append(time)
+    data_list[idx][1].append(value)
+    return data_list
+
+
+if __name__ == "__main__":
+    
+    fed = h.helicsCreateValueFederateFromConfig("transmission_config.json")
```
Comment (nightlark) [inline distribution1/dirtrbution.py]: Should this be loading the `distribution_config.json` file instead?

```suggestion
    fed = h.helicsCreateValueFederateFromConfig("distribution_config.json")
```

Comment (AadilLatif) [reply distribution1/dirtrbution.py]: fixed

**Comment by AadilLatif** [comment]
> It looks like the python script file name has a typo -- `dirtrbution.py` instead of `distribution.py`

fixed moved files to distribution 2 folder

**Review by nightlark — APPROVED** [review]

**Pull request merged**